index.html

```diff
@@ -4,9 +4,167 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
     <title>Douglas' Portifolio page</title>
 </head>
 <body>
-    <h2>Portifolio</h2>
+<div class="main-layout">
+
+    <nav>
+        <span>Luzinda Douglas</span> 
+        <img src="https://img.icons8.com/ios-glyphs/30/ffffff/menu--v1.png" alt="Hamburger menu icon">  
+    </nav>  
+
+    <section class="intro-section">
```

Comment:
Kindly add the background image to this section as shown below 😃

Your design:
![image](https://user-images.githubusercontent.com/73714615/186862536-5ad2a78c-dddd-46d2-85fa-4047ce1484ba.png)

Figma Design:
![image](https://user-images.githubusercontent.com/73714615/186862602-952fb534-dcaf-4b96-9f78-93410701cec9.png)